chore(docs): Migrate the callouts to bullets (#3580)
kroxylicious/kroxylicious: kroxylicious-docs/docs/_modules/configuring/con-configuration-outline.adoc

.Basic outline of a Kroxylicious configuration
[source,yaml]
----
filterDefinitions:
  - name: example
    type: org.example.filter.Example
    config:
      # ...
defaultFilters:
  - example
virtualClusters:
  - name: my-cluster-proxy
    targetCluster:
      # ...
    gateways:
    - name: ...
      # ...
    subjectBuilder:
      # ...

# ...
----
where:

* `filterDefinitions` defines the list of filter definitions. Each filter definition comprises:
** A `name` which identifies the filter definition. Name is required. It must be unique amongst all the other filters in the list.
** A `type` that specifies the filter implementation class. Required.
** `config` provides configuration to the filter. Usually required for non-trivial filters.
   The configuration required depends on the filter implementation.
* `defaultFilters` defines an ordered list of filters that the traffic sent to the virtual cluster will pass through.

>   It's possible to override this list at the virtual cluster level.

Reviewer: ```suggestion
  You can override this list at the virtual cluster level.
```
